src/components/project-pages/EroticStories.tsx

```diff
@@ -1,33 +1,128 @@
 import React from "react";
 import styled from "styled-components";
-import Timer from "../Timer";
-import { FlexboxProps, flexbox, layout, LayoutProps } from "styled-system";
-import statue from "../assets/project-page/statue.png";
+import { FlexboxProps, flexbox, layout, LayoutProps, grid, GridProps, space, SpaceProps, typography, TypographyProps } from "styled-system";
+import EroticStoriesPreview from "./EroticStoriesPreview";
+import alasImg from "./../assets/erotic-stories/alas-bateando.jpg";
+import ombligosImg from "./../assets/erotic-stories/ombligos.jpg";
+import { useTranslation } from "react-i18next";
 
-const Main = styled.main<FlexboxProps>`
-  display: flex;
-  height: 100vh;
-  overflow: hidden;
-  flex-direction: column;
-  ${flexbox}
+
+const Main = styled.main<TypographyProps>`
+  ${typography};
+  height: 100%;
 `;
 
-const Container = styled.div<FlexboxProps>`
-  display: flex;
+const Grid = styled.div<GridProps & FlexboxProps & LayoutProps>`
+  ${grid};
   ${flexbox};
+  ${layout};
 `;
 
-const Img = styled.img<LayoutProps>`
-  ${layout};
+const H1 = styled.h1<TypographyProps & SpaceProps>`
+  text-transform: uppercase;
+  ${space};
+  ${typography};
+`;
+
+const H2 = styled.h1<TypographyProps & SpaceProps>`
+  ${space};
+  ${typography};
+`;
+
+
+const EroticImageContainer = styled.div<GridProps & SpaceProps>`
+  ${grid};
+  ${space};
+  object-fit: contain;
+  height: fit-content;
+`;
+
+const EroticImage = styled.img<SpaceProps>`
+height: 100%;
+width: 100%;
+${space};
+`;
+
+const ScrollableTextContainer = styled.div<GridProps & SpaceProps>`
+  ${grid};
+  ${space};
+  object-fit: contain;
+  overflow: scroll;
+`;
+
+
+const ScrollableText = styled.p`
+  overflow: scroll;
 `;
 
+
+interface EroticStoryProps {
+  title: string;
+  img: string;
+  text: string;
+  alt: string;
+}
+
+
+
+
 const EroticStories = () => {
+  const { t } = useTranslation();
+  const eroticStories: EroticStoryProps[] = [
+    {
+      title: "Ombligos",
```

Comment:
where does the title get used?

Reply:
Ah I should have deleted that. It was meant to be the `key` for the map function which I was using originally.